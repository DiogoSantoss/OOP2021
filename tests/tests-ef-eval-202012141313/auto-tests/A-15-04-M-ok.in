9
1
20
0
0
