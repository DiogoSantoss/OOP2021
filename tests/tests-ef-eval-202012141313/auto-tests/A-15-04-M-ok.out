Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Consultas
1 - Produtos com Preço Abaixo de Limite
2 - Facturas Pagas por Cliente
0 - Sair
Escolha uma opção: Preço: BOX|1C6H5OH|P1|2|20|100|NORMAL
BOX|2H2|P2|4|20|100|AIR
BOX|3O3|P2|8|20|100|EXPRESS
BOX|4CO2|P2|16|20|0|PERSONAL
CONTAINER|6M2|P2|4|20|100|AIR|C4
CONTAINER|7M5|P2|8|20|100|EXPRESS|B4
BOOK|B1937|P1|5|2|5|How to fix almost everything|Satoshi Yamada|1928374650
Consultas
1 - Produtos com Preço Abaixo de Limite
2 - Facturas Pagas por Cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 